# =============================================================================
# Initial Pose Configuration for ROBOTIS OP3
# =============================================================================
# Semua nilai dalam RADIAN
# Pose ini adalah "bringup pose" - robot berdiri siap

initial_pose:
  # Movement time (seconds)
  move_time: 2.0

  # Joint positions (radians)
  # Konversi dari derajat: rad = deg * pi / 180
  
  joints:
    # Right Arm
    r_sho_pitch: 0.2618    # 15 deg
    r_sho_roll: -0.7854    # -45 deg  
    r_el: 0.7854           # 45 deg
    
    # Left Arm
    l_sho_pitch: -0.2618   # -15 deg
    l_sho_roll: 0.7854     # 45 deg
    l_el: -0.7854          # -45 deg
    
    # Right Leg
    r_hip_yaw: 0.0
    r_hip_roll: 0.0
    r_hip_pitch: 1.2217    # 70 deg
    r_knee: -2.4784        # -142 deg
    r_ank_pitch: -1.2217   # -70 deg
    r_ank_roll: 0.0
    
    # Left Leg
    l_hip_yaw: 0.0
    l_hip_roll: 0.0
    l_hip_pitch: -1.2217   # -70 deg
    l_knee: 2.4784         # 142 deg
    l_ank_pitch: 1.2217    # 70 deg
    l_ank_roll: 0.0
    
    # Head
    head_pan: 0.0
    head_tilt: -0.1745     # -10 deg
